modules/composer-service-api/src/main/java/org/opencastproject/composer/api/ComposerService.java

```diff
@@ -43,6 +43,11 @@
   String AUDIO_ONLY = "a";
   String VIDEO_ONLY = "v";
 
+  /** sourceAudioName options for composite - use one or both, if null is passed, both will be used */
```

Comment:
Hmm... why do you allow _null_? You can expect clients to correctly choose from "lower", "upper" or "both".